core/src/main/java/com/github/mizool/core/exception/AbstractUnprocessableEntityException.java

```diff
@@ -0,0 +1,29 @@
+package com.github.mizool.core.exception;
+
+/**
+ * Thrown when the entity is syntactically correct (e.g. all non-null properties are set), but could not be processed
+ * for semantic reasons (e.g. invalid references to other database entities).
+ * Extend this class to create an exception class specific to the respective problem.
+ */
+public abstract class AbstractUnprocessableEntityException extends RuntimeException
+{
+    public AbstractUnprocessableEntityException()
```

Comment:
Please fix [the SonarCloud issue](https://sonarcloud.io/project/issues?id=com.github.mizool%3Amizool&issues=AXtyA9xirRq0z5t8KDsJ&open=AXtyA9xirRq0z5t8KDsJ&pullRequest=195) for this and the other constructors (protected instead of public).